R/hp_spacyr.R

```diff
@@ -33,25 +85,25 @@ spacyr_df <- function(txt, ...) {
       "dep_",
       "is_currency",
       "like_url",
-      "like_email"
+      "like_email",
+      additional_attributes
     ),
     ...
   )
 }
 
-spacyr_attributes <- function() c("ent_type_", "pos", "is_currency", "like_url", "like_email")
 
-spacyr_default <- function() {
-  list(
-    type = "spacyr",
-    parent_id = "head_token_id",
-    child_id = "token_id",
-    child = "token",
-    node_type = "dep_",
-    link = "dep_",
-    root_tag = "ROOT",
-    attributes = spacyr_attributes())
-}
+
+spacyr_default_cols <- function() c("doc_id", "sentence_id", "token_id", "token",
+                                "lemma", "pos", "tag", "head_token_id", "dep_rel",
+                                "entity", "ent_type_", "dep_", "is_currency", "like_url", "like_email")
+
+#' Default settings for hp_spacyr attributes
+#'
+#' @export
+spacyr_default_attributes <- function() c("ent_type_", "pos_",
```

Comment:
Do these default attributes or columns ever change? I am on board with people adding stuff, but if they cannot remove stuff is there a need for these functions?

Reply:
I kept it as a function just in case if people want to keep all the existing tags without having to retype everything/remember what they are. If you think the function is not necessary I can remove!

Reply:
Ohhh I see. In that case, I think it’s fine to keep!